.openpublishing.publish.config.json

```diff
@@ -11,7 +11,12 @@
       "type_mapping": {
         "Conceptual": "Content",
         "ManagedReference": "Content",
-        "RestApi": "Content"
```

Comment:
If it's for SDP change you should remove `ManagedReference` too.

Also, you can delete the following pieces:
```
      "customized_template_paths": [
        "_dependentPackages/memberpage.plugins/content"
      ],

          {
      "id": "memberpage.plugins",
      "nuget_feed": "https://www.myget.org/F/docfx/api/v2",
      "path_to_root": "_dependentPackages/memberpage.plugins",
      "target_framework": "net45",
      "version": "latest"
    }
```